package.json

```diff
@@ -48,10 +48,11 @@
   },
   "homepage": "https://github.com/apollostack/graphql-tools#readme",
   "dependencies": {
-    "apollo-utilities": "^1.0.1",
     "apollo-link": "^1.0.0",
+    "apollo-utilities": "^1.0.1",
     "deprecated-decorator": "^0.1.6",
     "graphql-subscriptions": "^0.5.6",
+    "lodash.merge": "^4.6.0",
```

Comment:
I don't think we need lodash.merge - we could just write a simple function that loops through a few objects and merges them. Shouldn't be more than 10-20 lines of code I think since we don't need to support all the stuff that lodash.merge does.

We've intentionally kept the lodash dependency out.

Reply:
There is https://github.com/apollographql/graphql-tools/blob/master/src/stitching/mergeSchemas.ts#L306-L322. You can extract it and use it for merging.